src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
```

Comment:
You can move the SHORT_TERM_DISCOUNT constant outside of the function, as it's a constant that doesn't change between function calls.

Reply:
> You can move the SHORT_TERM_DISCOUNT constant outside of the function, as it's a constant that doesn't change between function calls.

Thanks, I've already fixed it